bin/build_container_image

```diff
@@ -3,7 +3,7 @@
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
 DEFAULT_TAG="latest"
-[ "$GITHUB_EVENT_NAME" == "push" -a "$GITHUB_REF_NAME" != "master" ] && \
+[[ "$GITHUB_EVENT_NAME" =~ ^(push|workflow_dispatch)$ && "$GITHUB_REF_NAME" != "master" ]] && \
```

Comment:
I think this is part of the reason that a manual build wasn't working correctly?